gstd/gstd.c

```diff
@@ -128,7 +133,7 @@ main (gint argc, gchar * argv[])
   }
 
   if (daemon || kill) {
-    if (!gstd_log_init (gstdlogfile, gstlogfile)) {
+    if (!gstd_log_init (gstdlogfile, gstlogfile, !nolog && !kill)) {
```

Comment:
This seems hard to read to me without comment. Should we separate the logic and create a boolean with the name `enabled` for readability and add that the logging is only enabled under certain conditions as a small comment?